osu.Game/Online/Chat/ChannelManager.cs

```diff
@@ -67,11 +68,33 @@ public class ChannelManager : PollingComponent, IChannelPostTarget
 
         public readonly BindableBool HighPollRate = new BindableBool();
 
+        private IBindable<bool> isIdle;
+
         public ChannelManager()
         {
             CurrentChannel.ValueChanged += currentChannelChanged;
+        }
 
-            HighPollRate.BindValueChanged(enabled => TimeBetweenPolls.Value = enabled.NewValue ? 1000 : 6000, true);
+        [BackgroundDependencyLoader]
+        private void load(IdleTracker idleTracker)
```

Comment:
As per CI, probably easiest to allow the tracker to be `null` and just keep chugging along presuming `isIdle = false` if it's not present.